src/presentation/tenant-usage-presenter-factory.ts

```diff
@@ -46,6 +51,7 @@ export class TenantUsagePresenterFactory {
     return new TablePresenter(
       costTableViewGenerator,
       this.tableFactory.buildMeshTable(),
+      stats,
```

Comment:
d: I try to keep arguments as little and clean as possible. Usually everything which does not change over the lifetime of an object I put into the ctor, the change-per-call stuff goes into the method. However here we have a bit of a special case. Currently, it does not change, however in the future it might be that you need to replace stats objects between calls (or it contains a mutable state so it can be re-used to fetch this data). 
But even worse we do two different things here, we put it in the presenter ctor but in the draw method. I know this was a bit unclear also in the original implementation as we struggled to decide is it a per-call data or per-object data blob. But I would like to clarify this.
Currently, I would tend to put it rather in the ctor and say it is reused, filled by the facades and if present and wired into the Presenter layer it will get used to display the runtime. You then in theory could invoke the draw logic in the presenter multiple times with different data outputs. What do you think?